src/Dafda/Consuming/MessageRegistration.cs

```diff
@@ -28,7 +28,7 @@ public sealed class MessageRegistration
         private static string EnsureValidTopicName(string topicName)
         {
             // Passing a null topic, will cause Confluent to throw an AccessViolationException, which cannot be caught by the service, resulting in a hard crash without logs.
-            if (string.IsNullOrWhiteSpace(topicName))
```

Comment:
Why should we allow empty string as a topic name?

Reply:
@cristian-dfds Currently there is a NoopHandler which subscribes with an empty string, and I didn't want to make too many changes to the existing framework.